roles/openshift_ovirt/defaults/main.yml

```diff
@@ -1,51 +1,51 @@
 ---
-# Defaults for oVirt/RHV provider integration
+# Defaults for oVirt/oVirt provider integration
```

Comment:
Here's a case where the sed did not make sense. It may need to be reduced to just oVirt

Reply:
Thanks, I updated